consoles/localXvnc.pm

```diff
@@ -96,7 +96,8 @@ sub activate {
     sleep 1;
 
     # we need a window manager for fullscreen apps to work
-    system("DISPLAY=$display icewm -c $bmwqemu::scriptdir/consoles/icewm.cfg & echo \"icewm PID is \$!\"");
+    $icewm_return_code = system("DISPLAY=$display icewm -c $bmwqemu::scriptdir/consoles/icewm.cfg & echo \"icewm PID is \$!\"");
+    die('sh: icewm: command not found') if ($icewm_return_code == 32512);
```

Comment:
You check for the exact return code `32512` here? I doubt that this return code will always be that arbitrary number. Why not checking `!= 0` instead?

Reply:
this is exit status 127 if you pass it through WEXITSTATUS :)

Reply:
maybe is safer to check if the binary is actually there before calling it or even  you can also use run_cmd https://github.com/os-autoinst/os-autoinst/search?q=runcmd&unscoped_q=runcmd

Reply:
$? == @127
But, system(...) == 32512

I didn't want to let it die on other cases.